### PR TITLE
Add words about IRODS_ENVIRONMENT_FILE to icd help text (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: src/icd.cpp [hunk 1]
```diff
@@ -146,14 +146,34 @@ checkColl( rcComm_t *Conn, char *path ) {
...
-    printf( "Usage: %s [-vh] [directory]\n", prog );
-    printf( "If no directory is entered, the cwd is set back to your home\n" );
-    printf( "directory as defined in your .rodsEnv file.\n" );
-    printf( "Like the unix 'cd', '..' will move up one level and \n" );
-    printf( "'/name' starts at the root.\n" );
-    printf( " -v  verbose\n" );
-    printf( " -V  very verbose\n" );
-    printf( " -h  this help\n" );
-    printReleaseInfo( "icd" );
+    printf( "Change the working collection.\n\n" );
```
Comment (alanking) [inline src/icd.cpp:149]:
```suggestion
    printf( "Change the current working collection.\n\n" );
```

File: src/icd.cpp [hunk 2]
```diff
@@ -146,14 +146,34 @@ checkColl( rcComm_t *Conn, char *path ) {
...
+    printf( "Usage: %s [-vVh] [COLLECTION]\n\n", prog );
+
+    printf( "If invoked without any arguments, the current working collection is set\n" );
+    printf( "back to your home collection as defined in your irods_environment.json\n" );
+    printf( "file.\n\n" );
+
+    printf( "If COLLECTION matches \"..\", the current working collection is set to.\n" );
+    printf( "the parent collection.\n\n" );
+
+    printf( "If COLLECTION begins with a forward slash, the path is treated as an.\n" );
```
Comment (alanking) [inline src/icd.cpp:160]:
```suggestion
    printf( "If COLLECTION begins with a forward slash, the path is treated as an\n" );
```

File: src/icd.cpp [hunk 3]
```diff
@@ -146,14 +146,34 @@ checkColl( rcComm_t *Conn, char *path ) {
...
-    printReleaseInfo( "icd" );
+    printf( "Change the working collection.\n\n" );
+
+    printf( "Usage: %s [-vVh] [COLLECTION]\n\n", prog );
+
+    printf( "If invoked without any arguments, the current working collection is set\n" );
+    printf( "back to your home collection as defined in your irods_environment.json\n" );
+    printf( "file.\n\n" );
+
+    printf( "If COLLECTION matches \"..\", the current working collection is set to.\n" );
```
Comment (alanking) [inline src/icd.cpp:157]:
```suggestion
    printf( "If COLLECTION matches \"..\", the current working collection is set to\n" );
```

**Review by alanking — COMMENTED** [review]
Seems good

File: src/icd.cpp (hunk 1 above)
Comment (korydraughn) [reply src/icd.cpp]: Done

File: src/icd.cpp (hunk 2 above)
Comment (korydraughn) [reply src/icd.cpp]: Done

File: src/icd.cpp (hunk 3 above)
Comment (korydraughn) [reply src/icd.cpp]: Done

**Review by alanking — APPROVED** [review]
I think this seems good. Please squash and # once an issue exists for this.

**Comment by korydraughn** [comment]
#'d

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**